server/sonar-server/src/main/java/org/sonar/ce/monitoring/CEQueueStatus.java

```diff
@@ -134,5 +84,5 @@
   /**
    * Time spent processing batch reports since startup.
    */
-  long getProcessingTime();
+  long getProcessingTimeInMs();
```

Comment:
What possible bug?
And what else could it be but milliseconds anyway?

"It's a time! It's a long! It's milliseconds!"

All other longs in this interface are milliseconds and there is no need to specify it anywhere else than in the Javadoc. Longs as milliseconds in (parameters), longs as milliseconds out.